<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAbsensi</class>
 <widget class="QWidget" name="FormAbsensi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ABSENSI KARYAWAN</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="rHadir">
      <property name="text">
       <string>Hadir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rIzin">
      <property name="text">
       <string>Izin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rSakit">
      <property name="text">
       <string>Sakit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="kdAbsen"/>
    </item>
    <item>
     <widget class="QLineEdit" name="nik"/>
    </item>
    <item>
     <widget class="QLineEdit" name="namaKry"/>
    </item>
    <item>
     <widget class="QDateEdit" name="tglrekap"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>160</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kode Absen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nik</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nama</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tanggal Rekap</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableAbsen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>411</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>kd_absen</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nik</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nama_karyawan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>tgl_rekap</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>status_kehadiran</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>381</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnDone">
      <property name="text">
       <string>Simpan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBersih">
      <property name="text">
       <string>BERSIH</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHapus">
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="editcari">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>229</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>158</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cari Absen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
